docs/concepts/services-networking/service.md

```diff
@@ -218,13 +218,13 @@ it's not installed kube-proxy will fall back to iptables proxy mode.
 
 ![Services overview diagram for ipvs proxy](/images/docs/services-ipvs-overview.svg)
 
-In any of proxy model, any traffic bound for the Service’s IP:Port is 
-proxied to an appropriate backend without the clients knowing anything 
-about Kubernetes or Services or Pods. Client-IP based session affinity 
-can be selected by setting service.spec.sessionAffinity to "ClientIP" 
-(the default is "None"), and you can set the max session sticky time by 
-setting the field service.spec.sessionAffinityConfig.clientIP.timeoutSeconds 
-if you have already set service.spec.sessionAffinity to "ClientIP" 
```

Comment:
My editor removed trailing whitespace, I can revert this if you don't want so many changes

Reply:
It's fine! 👍

Reply:
While we're here, though, it'd be good to replace this paragraph to make some minor fixes:
```
In any of these proxy models, any traffic bound for the Service’s IP:Port is proxied to an appropriate backend without the clients knowing anything about Kubernetes or Services or Pods. Client-IP based session affinity can be selected by setting `service.spec.sessionAffinity` to "ClientIP" (the default is "None"), and you can set the max session sticky time by setting the field `service.spec.sessionAffinityConfig.clientIP.timeoutSeconds` if you have already `set service.spec.sessionAffinity` to "ClientIP" (the default is “10800”).
```
So:
> In any of these proxy models, any traffic bound for the Service’s IP:Port is proxied to an appropriate backend without the clients knowing anything about Kubernetes or Services or Pods. Client-IP based session affinity can be selected by setting `service.spec.sessionAffinity` to "ClientIP" (the default is "None"), and you can set the max session sticky time by setting the field `service.spec.sessionAffinityConfig.clientIP.timeoutSeconds` if you have already `set service.spec.sessionAffinity` to "ClientIP" (the default is “10800”).